tao_compiler/mlir/disc/transforms/disc_transform_legalize_to_loop.cc

```diff
@@ -132,10 +141,26 @@ LogicalResult DiscTransformLegalizeToLoopPass::outlineFusionOp(
 
 LogicalResult DiscTransformLegalizeToLoopPass::runTransformPipeline(
     ModuleOp m) {
+  if (VLOG_IS_ON(1))
```

Comment:
looks wired with multiple loggers together. :P

Reply:
LLVM community does not support logging control in release build, which is very helpful for developers. Thus . Thus I think it's worth to introduce this feature here.